src/System.Text.Encoding/tests/Fallback/EncoderReplacementFallbackTests.cs

```diff
@@ -104,13 +104,23 @@ public void CreateFallbackBuffer_Fallback_Char_Char(string replacement, bool exp
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
         public void CreateFallbackBuffer_Fallback_InvalidSurrogateChars_ThrowsArgumentOutOfRangeException()
         {
             EncoderFallbackBuffer buffer = new EncoderReplacementFallback().CreateFallbackBuffer();
 
             Assert.Throws<ArgumentOutOfRangeException>("charUnknownHigh", () => buffer.Fallback('a', '\uDC00', 0));
-            Assert.Throws<ArgumentOutOfRangeException>(
-                RuntimeDetection.IsNetFramework ? "CharUnknownLow" : "charUnknownLow", () => buffer.Fallback('\uD800', 'a', 0));
```

Comment:
On the bright side we get to remove `RuntimeDetection` so the net increase in lines isn't *too* bad.